### PR TITLE
Update il.m3u

### PR DESCRIPTION
**Pull request by freearhey**

### CODE REVIEW

File: streams/il.m3u [hunk 1]
```diff
@@ -17,19 +17,19 @@ https://edge3.uk.kab.tv/live/tv66-heb-high/playlist.m3u8
...
-#EXTINF:-1 tvg-id="" status="online",Hidabroot (576p) [Not 24/7]
+#EXTINF:-1 tvg-id="Hidabroot.il" status="online",Hidabroot (576p) [Not 24/7]
 https://stream71.shidur.net/htvlive2/smil:live2.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="online",Hidabroot (576p)
+#EXTINF:-1 tvg-id="Hidabroot.il" status="online",Hidabroot (576p)
 https://stream72.shidur.net/htvlive2/smil:live2.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="online",Musayof (Israel) (240p) [Not 24/7]
 http://wowza.media-line.co.il/Musayof-Live/livestream.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="blocked",As a Child (720p)
+#EXTINF:-1 tvg-id="KanEducational.il" status="blocked",As a Child (720p)
```
Comment (UltraHDR) [inline streams/il.m3u:30]: IS this correct?

**Review by UltraHDR — DISMISSED** [review]

File: streams/il.m3u (hunk 1 above)
Comment (freearhey) [reply streams/il.m3u]: I wasn't able to run the broadcast itself, so this is just my guess based on the link. But it's really better to leave it blank for now.

**Review by UltraHDR — DISMISSED** [review]

File: streams/il.m3u (hunk 1 above)
Comment (UltraHDR) [reply streams/il.m3u]: That link doesn't work for me with a Israel VPN. The new link is

https://www.kankids.org.il/
```
#EXTINF:-1 tvg-id="KanEducational.il" status="online",Kan Educational (720p)
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/x7ymw7f
```

File: streams/il.m3u [hunk 2]
```diff
@@ -33,7 +37,7 @@ http://wowza.media-line.co.il/Musayof-Live/livestream.sdp/playlist.m3u8
 https://d18b0e6mopany4.cloudfront.net/out/v1/08bc71cf0a0f4712b6b03c732b0e6d25/index.m3u8
 #EXTINF:-1 tvg-id="" status="blocked",As a Child (720p)
 https://kanlivep2event-i.akamaihd.net/hls/live/747600/747600/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="blocked",KAN 11 Israel (360p) [Geo-blocked]
+#EXTINF:-1 tvg-id="Kan11.il" status="blocked",KAN 11 Israel (360p) [Geo-blocked]
 https://kanlivep2event-i.akamaihd.net/hls/live/747610/747610/master.m3u8
```
Comment (UltraHDR) [inline streams/il.m3u:41]: It's geo-blocked
https://www.dailymotion.com/video/x7wjmog

Comment (UltraHDR) [inline streams/il.m3u:41]:
```suggestion
https://kan11.media.kan.org.il/hls/live/2024514/2024514/master.m3u8
```

File: streams/il.m3u [hunk 3]
```diff
@@ -33,7 +37,7 @@ http://wowza.media-line.co.il/Musayof-Live/livestream.sdp/playlist.m3u8
 https://d18b0e6mopany4.cloudfront.net/out/v1/08bc71cf0a0f4712b6b03c732b0e6d25/index.m3u8
 #EXTINF:-1 tvg-id="" status="blocked",As a Child (720p)
 https://kanlivep2event-i.akamaihd.net/hls/live/747600/747600/playlist.m3u8
```
Comment (UltraHDR) [inline streams/il.m3u:39]:
```suggestion
https://kan23.media.kan.org.il/hls/live/2024691/2024691/master.m3u8
```

File: streams/il.m3u (hunk 1 above)
Comment (UltraHDR) [reply streams/il.m3u]: Found a m3u8 alternative

File: streams/il.m3u [hunk 4]
```diff
@@ -33,7 +37,7 @@ http://wowza.media-line.co.il/Musayof-Live/livestream.sdp/playlist.m3u8
 https://d18b0e6mopany4.cloudfront.net/out/v1/08bc71cf0a0f4712b6b03c732b0e6d25/index.m3u8
 #EXTINF:-1 tvg-id="" status="blocked",As a Child (720p)
```
Comment (UltraHDR) [inline streams/il.m3u:38]:
```suggestion
#EXTINF:-1 tvg-id="KanEducational.il" status="online",Kan Kids (720p)
```

**Review by UltraHDR — DISMISSED** [review]

File: streams/il.m3u [hunk 5]
```diff
@@ -33,7 +37,7 @@ http://wowza.media-line.co.il/Musayof-Live/livestream.sdp/playlist.m3u8
...
 https://kanlivep2event-i.akamaihd.net/hls/live/747600/747600/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="blocked",KAN 11 Israel (360p) [Geo-blocked]
+#EXTINF:-1 tvg-id="Kan11.il" status="blocked",KAN 11 Israel (360p) [Geo-blocked]
 https://kanlivep2event-i.akamaihd.net/hls/live/747610/747610/master.m3u8
-#EXTINF:-1 tvg-id="" status="timeout",ch 21 ערוץ הקניות (360p)
+#EXTINF:-1 tvg-id="TheShoppingChannel.il" status="timeout",ch 21 ערוץ הקניות (360p)
 http://82.80.192.30/shoppingil_ShoppingIL21TVRepeat/smil:ShoppingIL21TV.smil/playlist.m3u8
```
Comment (UltraHDR) [inline streams/il.m3u:43]:
```suggestion
https://shoppingil-rewriter.vidnt.com/index.m3u8
```

**Review by UltraHDR — DISMISSED** [review]

**Review by UltraHDR — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**